rasa/core/events/__init__.py

```diff
@@ -425,7 +426,7 @@ def __eq__(self, other):
             return (self.key, self.value) == (other.key, other.value)
 
     def as_story_string(self):
-        props = json.dumps({self.key: self.value})
+        props = json.dumps({self.key: self.value}, ensure_ascii=False)
```

Comment:
In your latest commits you use the `json_to_string`, maybe it should be used here too? 
I don't think it matters much but I think we should either use that function everywhere its functionality appears or just not have it :P

Reply:
Nah, i tried it, it's more for outputting to `.json` files since `json_to_string` puts stuff on new lines and indents them, which we don't want in generated stories.

Reply:
:+1: